.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# nvidia/cuda:11.4.1-cudnn8-devel-ubuntu20.04
```

Comment:
file not needed

Reply:
Ideally I'd like to use https://github.com/microsoft/vscode-dev-containers/pull/1176 cause the current devcontainer only supports CUDA on windows through WSL2